docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,358 @@
+---
+title: "WebLogic Domain"
+date: 2019-10-01T14:32:31-05:00
+weight: 2
+description: "WebLogic domains and operator's logs"
+---
+
+
+#### Overview
+This document describes to how to configure a WebLogic domain to use Fluentd to send log information to Elasticsearch.
+
+Here's the general mechanism for how this works:
+
+* `fluentd` runs as a separate container in the Administration Server and Managed Server pods
+* The log files reside on a volume that is shared between the `weblogic-server` and `fluentd` containers
+* `fluentd` tails the domain logs files and exports them to Elasticsearch
+* A `ConfigMap` contains the filter and format rules for exporting log records
+
+##### Sample code
+
+The samples in this document assume an existing domain is being edited.  However, all changes to the domain YAML can be performed before the domain is created.
+
+For sample purposes, this document will assume a domain with the following attributes is being configured:
+
+* Domain name is `bobs-bookstore`
+* Kubernetes namespace is `bob`
+* Kubernetes secret is `bobs-bookstore-weblogic-credentials`
+
+The sample Elasticsearch configuration is:
+```text
+    elasticsearchhost: elasticsearch.bobs-books.sample.com
+    elasticsearchport: 443
+    elasticsearchuser: bob
+    elasticsearchpassword: changeme
+```
+
+#### Configure log files to use a volume
+The domain log files must be written to a volume that can be shared between the `weblogic-server` and `fluentd` containers.  The following elements are required to accomplish this:
+
+* `logHome` must be a path that can be shared between containers
+* `logHomeEnabled` must be set to `true` so that the logs will be written outside the pod and persist across pod restarts
+* A `volume` must be defined that the log files will reside on.  In the example `emptyDir` is a volume that gets created empty when a Pod is created.  It will persist across pod restarts but deleting the pod would delete the `emptyDir` content.
+* The `volumeMounts` mounts the named volume created with `emptyDir` and establishes the base path for accessing the volume.
+
+**NOTE**: For brevity only the paths to the relevant configuration being added is shown.  A complete example of a domain definition is at the end of this document.
```

Comment:
For brevity -> For brevity, (comma; make this change globally, in every instance of this note)